src/main/java/io/percy/appium/lib/PercyOptions.java

```diff
@@ -42,6 +42,20 @@ public Boolean setPercyIgnoreErrors() {
         return true;
     }
 
+    public Boolean setPercyFullPageScreenshotEnabled() {
```

Comment:
not sure why we are addinig percyOption here? we will already have a screenshotType option set by user.